packages/react-i18n/README.md

```diff
@@ -427,6 +427,46 @@ generateTranslationIndexes();
 webpack(require(./webpack.config.js));
 ```
 
+#### Statically embedding locale-specific translations
+
+For applications large applications, even asynchronously loaded translations can significantly degrad the user experience:
+
+- Bundlers like webpack have to embed kilobytes of data to track each translation import
+- Users not using the "default" language have to download kilobytes of translations for every language
+
+To avoid this, it is possible to build versions of app with specific locale translations embedded directly in JavaScript. To achieve this, run the Babel plugin in `from-dictionary-index` mode:
+
+```js
+// babel.config.js
+{
+  plugins: [
+    ['@shopify/react-i18n/babel', {mode: 'from-dictionary-index'}],
+  ],
+}
+```
+
+Then generate `translations/index.js` files containing specific locale data using the `@shopify/react-i18n/generate-dictionaries` helper. e.g., the following code generates three versions of an application with English, French, and German content using webpack.
+
+```js
+// generate-translations.js
+const {
+  generateTranslationDictionaries,
+} = require('@shopify/react-i18n/generate-dictionaries');
+
+// Build English app.
+await generateTranslationDictionaries(['fr']);
+webpack(require(./webpack.config.js));
+
+// Build French app.
+await generateTranslationDictionaries(['fr'], {fallbackLocale: 'en'});
+webpack(require(./webpack.config.js));
```

Comment:
I know we don't show it in the old example but it feels really weird to me that we show this code and the babel config but not what a simple webpack config would look like for this example.

Reply:
I've added one, but valid, minimal webpack configs are _hard_.  I think what I've added is a dirty lie, but I guess it could help someone get started 😄